lib/postgrex/query.ex

```diff
@@ -20,7 +20,8 @@ defmodule Postgrex.Query do
     result_oids:    [Postgrex.Types.oid] | nil,
     result_formats: [:binary | :text] | nil,
     result_types:   [Postgrex.Types.type] | nil,
-    types:          Postgrex.Types.state | nil}
+    types:          Postgrex.Types.state | nil
+  }
```

Comment:
Maybe drop this change since it only a style change in this file but does not fix the styling completely :)